### DISCUSSION TITLE
Any plans to add a Sidebar component?

### DISCUSSION THREAD

**Discussion by Dann1y** [tremorlabs/tremor-npm#385] [Q&A] [answered]
I didn't find a Sidebar component in [documentation.](https://www.tremor.so/docs/getting-started/installation)

I think it would be very helpful to make Sidebar, because many users of the dashboard, including me, make dashboard with sidebar.

If it's okay, I'll make one and PR on it.

Thanks for your good project!

**Comment by mitrotasios** [comment, marked as answer]
hey, a sidebar is not planned for now, we'd like to focus on the more essential components in the beginning :) We will add one eventually

**Comment by rockse** [comment]
I would like to have a sidebar as it is a traditional component that all users use to navigate. Having it built-in would be greatly appreciated.
